cmd/manager/suitererunner.go

```diff
@@ -73,16 +78,40 @@ func RerunSuite(cmd *cobra.Command, args []string) {
 	fmt.Printf("Got %d scans from the ComplianceSuite '%s'\n", len(scans.Items), conf.Name)
 
 	for _, scan := range scans.Items {
-		scanCopy := scan.DeepCopy()
-		if scanCopy.Annotations == nil {
-			scanCopy.Annotations = make(map[string]string)
-		}
-		scanCopy.Annotations[compv1alpha1.ComplianceScanRescanAnnotation] = ""
+		currentScan := &scan
+		key := types.NamespacedName{Name: scan.GetName(), Namespace: scan.GetNamespace()}
+		err := backoff.Retry(func() error {
+			var scanCopy *compv1alpha1.ComplianceScan
+			if currentScan == nil {
```

Comment:
I think we will need to fetch again regardless of the error from the previous attempt. In the non-expire/conflict case the version of the resource we previously got from the list would be old still.